src/nwb2bids/_tools/_pluralize.py

```diff
@@ -0,0 +1,7 @@
+def _pluralize(n: int, word: str, plural: str | None = None) -> str:
+    if n == 1:
+        return word
+    else:
+        if plural is None:
+            plural = word + "s"
+        return plural
```

Comment:
```suggestion
    if n == 1:
        return word
    elif plural is None:
        return word + "s"
    else:
        return plural
```

is IMHO easier to follow and avoids overload of plural